StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentMethodCollectionView.swift

```diff
@@ -91,11 +103,29 @@ extension SavedPaymentMethodCollectionView {
             button.accessibilityLabel = String.Localized.edit
             return button
         }()
+        lazy var defaultBadge: UILabel = {
+            let label = UILabel()
+            label.font = appearance.scaledFont(for: appearance.font.base.medium, style: .footnote, maximumPointSize: 20)
+            label.textColor = appearance.colors.textSecondary
+            label.adjustsFontForContentSizeCategory = true
+            label.text = STPLocalizedString("Default", "Label for identifying the default payment method.")
```

Comment:
Let's move this into a variable into `StripePaymentSheet/StripePaymentSheet/Source/Categories/String+Localized.swift` and update the usage in LinkPaymentMethodPicker-Cell. This way we can easily reuse the translations w/o worrying about matching the comment on it.